### PR TITLE
Adding new conversation types

### PR DESCRIPTION
**Pull request by RamjotSingh**

### CODE REVIEW

File: api-reference/beta/resources/anonymousguestconversationmember.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Represents anonymous guest in a chat. Anonymous users do not have Microsoft Teams identity and can join meetings using meeting join links. You can read more about
+anonymous users [here](/microsoftteams/non-standard-users#anonymous-users).
+
+
+Inherits from [conversationMember](../resources/conversationmember.md).
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|anonymousGuestId|String|Unique Id representing the user. **Note:** This Id can change if the user leaves the meeting and join again or joins from a different device.|
```
Comment (AkJo) [inline api-reference/beta/resources/anonymousguestconversationmember.md:25]: joins

File: api-reference/beta/resources/anonymousguestconversationmember.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+|displayName|String|Name provided by the user when joining meeting. Inherited from [conversationMember](../resources/conversationmember.md).|
```
Comment (AkJo) [inline api-reference/beta/resources/anonymousguestconversationmember.md:26]: joining the meeting

File: api-reference/beta/resources/anonymousguestconversationmember.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+|id|String|Membership Id representing this resource. Inherited from [entity](../resources/entity.md).|
```
Comment (AkJo) [inline api-reference/beta/resources/anonymousguestconversationmember.md:27]: Is this part necessary?

File: api-reference/beta/resources/microsoftaccountuserconversationmember.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+doc_type: resourcePageType
+---
+
+# microsoftAccountUserConversationMember resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents a Microsoft personal account user in a chat.
```
Comment (AkJo) [inline api-reference/beta/resources/microsoftaccountuserconversationmember.md:16]: Wondering if the casing should be Microsoft Personal Account user in a chat.

or this probably needs to be rephrased to 

Represents a user with a personal Microsoft account in a chat.

File: api-reference/beta/resources/microsoftaccountuserconversationmember.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+
+Inherits from [conversationMember](../resources/conversationmember.md).
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|displayName|String|Display name of the user. Inherited from [conversationMember](../resources/conversationmember.md).|
+|id|String|Membership Id representing this resource. Inherited from [entity](../resources/entity.md).|
+|roles|String collection|Special roles for this user. Inherited from [conversationMember](../resources/conversationmember.md).|
```
Comment (AkJo) [inline api-reference/beta/resources/microsoftaccountuserconversationmember.md:26]: I am noticing that we are listing out the possible values for these roles neither here nor in the conversationMember file.

File: api-reference/beta/resources/skypeforbusinessuserconversationmember.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+doc_type: resourcePageType
+---
+
+# skypeForBusinessUserConversationMember resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents a Skype for business user in a chat. You can learn more about Teams and Skype for business interoperability [here](/microsoftteams/teams-and-skypeforbusiness-coexistence-and-interoperability).
```
Comment (AkJo) [inline api-reference/beta/resources/skypeforbusinessuserconversationmember.md:16]: Skype For Business user in a chat.

Comment (AkJo) [inline api-reference/beta/resources/skypeforbusinessuserconversationmember.md:16]: Skype For Business

File: api-reference/beta/resources/skypeforbusinessuserconversationmember.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+Inherits from [conversationMember](../resources/conversationmember.md).
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|displayName|String|Display name of the user. Inherited from [conversationMember](../resources/conversationmember.md).|
+|id|String|Membership Id representing this resource. Inherited from [entity](../resources/entity.md).|
+|roles|String collection|Special roles for this user. Inherited from [conversationMember](../resources/conversationmember.md).|
+|tenantId|String|Id of the tenant the user belongs to.|
```
Comment (AkJo) [inline api-reference/beta/resources/skypeforbusinessuserconversationmember.md:27]: "Id of the tenant that the user belongs to. "?

File: api-reference/v1.0/api/chat-post.md [hunk 8]
```diff
@@ -19,7 +19,7 @@ One of the following permissions is required to call this API. To learn more, in
 |:---|:---|
 |Delegated (work or school account)|Chat.Create, Chat.ReadWrite|
 |Delegated (personal Microsoft account) | Not supported. |
-|Application | Not supported. |
+|Application | Chat.Create |
```
Comment (AkJo) [inline api-reference/v1.0/api/chat-post.md:22]: Are we lighting up V1 at the same time?

**Review by AkJo — COMMENTED** [review]
Did an initial pass and left feedback,

File: api-reference/beta/resources/anonymousguestconversationmember.md (hunk 3 above)
Comment (RamjotSingh) [reply api-reference/beta/resources/anonymousguestconversationmember.md:27]: Which part? The note part?

File: api-reference/beta/resources/microsoftaccountuserconversationmember.md (hunk 4 above)
Comment (RamjotSingh) [reply api-reference/beta/resources/microsoftaccountuserconversationmember.md:16]: Changed to personal Microsoft account

File: api-reference/beta/resources/microsoftaccountuserconversationmember.md (hunk 5 above)
Comment (RamjotSingh) [reply api-reference/beta/resources/microsoftaccountuserconversationmember.md:26]: This is auto generated docs so I suppose they don't want us mentioning.

File: api-reference/beta/resources/anonymousguestconversationmember.md (hunk 3 above)
Comment (RamjotSingh) [reply api-reference/beta/resources/anonymousguestconversationmember.md:27]: Discussed offline.

**Review by AkJo — APPROVED** [review]

**Review by Lauragra — APPROVED** [review]

**Pull request merged**